tests/unit-tests/python/foglamp_test/services/core/api/test_statistics.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import asyncpg
+import requests
+import pytest
+
+__author__ = "Praveen Garg"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+__DB_NAME = "foglamp"
+BASE_URL = 'http://localhost:8081/foglamp'
+headers = {"Content-Type": 'application/json'}
+
+pytestmark = pytest.mark.asyncio
+
+
+async def add_statistics_test_data():
+    conn = await asyncpg.connect(database=__DB_NAME)
```

Comment:
@izoratti - you started to discuss ways of using the database in the unit tests that did not directly access it like this. We need to move that forward as we are getting tests making direct access like this and will need to go back and change then when the generic solution is implemented.